js/foundation.dropdownMenu.js

```diff
@@ -156,9 +156,20 @@ class DropdownMenu extends Plugin {
           if ($elem.attr('data-is-click') === 'true' && _this.options.clickOpen) { return false; }
 
           clearTimeout($elem.data('_delay'));
-          $elem.data('_delay', setTimeout(function() {
+          $elem.data('_delay', setTimeout(function () {
+
+            // Ignore "magic mouseleave": when the mouse simply disapear from the document
+            // (like when entering in browser input suggestions See https://git.io/zf-11410),
+            // unless we actually leaved the window (and document lost focus).
```

Comment:
Thanks :+1: I think you mean to say `unless we actually left the window` here, but otherwise, looks good.